### PR TITLE
New package: libslirp

### PR DESCRIPTION
**Pull request by ifurther**
#7118

### CODE REVIEW

File: packages/libslirp/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.freedesktop.org/slirp/libslirp
+TERMUX_PKG_DESCRIPTION="A general purpose TCP-IP emulator."
+TERMUX_PKG_LICENSE="BSD 3-Clause"
+TERMUX_PKG_LICENSE_FILE="COPYRIGHT"
```
Comment (Grimler91) [inline packages/libslirp/build.sh:4]: COPYRIGHT is in the [list of names](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L43) that are search for [BSD 3-Clause](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_install_license.sh#L36), so should be no need to specify TERMUX_PKG_LICENSE_FILE

File: packages/libslirp/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=4.6.1
+TERMUX_PKG_SRCURL=https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v${TERMUX_PKG_VERSION}/libslirp-v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=69ad4df0123742a29cc783b35de34771ed74d085482470df6313b6abeb799b11
+TERMUX_PKG_DEPENDS="glib, valgrind"
```
Comment (Grimler91) [inline packages/libslirp/build.sh:9]: Pretty sure valgrind is not a dependency (?). If it enhances libslirp then perhaps add it to TERMUX_PKG_SUGGESTS or TERMUX_PKG_RECOMMENDS

Comment (xeffyr) [reply packages/libslirp/build.sh:9]: Valgrind is not a build or run time dependency of libslirp and should be removed.

File: packages/libslirp/build.sh (hunk 1 above)
Comment (ifurther) [reply packages/libslirp/build.sh]: I updated it

File: packages/libslirp/build.sh (hunk 2 above)
Comment (ifurther) [reply packages/libslirp/build.sh]: Sorry, I remvoed it

**Review by xeffyr — APPROVED** [review]

**Pull request merged**